src/transformers/models/aria/modeling_aria.py

```diff
@@ -1193,10 +1194,11 @@ def forward(
                 config.vocab_size]` or -100 (see `input_ids` docstring). Tokens with indices set to `-100` are ignored
                 (masked), the loss is only computed for the tokens with labels in `[0, ..., config.vocab_size]`.
 
-            num_logits_to_keep (`int`, *optional*):
-                Calculate logits for the last `num_logits_to_keep` tokens. If `0`, calculate logits for all
+            num_logits_to_keep (`int` or `torch.Tensor`, *optional*):
```

Comment:
would be nice to add the proper shape of the tensor, here you only allow 1d tensor, could be just a random index to keep tensor